### DISCUSSION TITLE
This repository has been disabled

### DISCUSSION THREAD

**Discussion by arun12341234** [community/community#31824] [Other Feature Feedback, Questions, & Ideas]
I left my GitHub personal token in the repository,
To fix it, I done,
1. remove the token from the repository
2. remove the personal token from my account too,

now, tokens are no longer accessible, 
now it become nothing more than string only,

But now I see the repository disabled,

when my organization people know that I will lose my job position,

please help it resolve the issue,

https://github.com/Jerry-Chae/plugins

**Comment by jfullstackdev** [comment]
seems GitHub disabled it, if that is by mistake you can try contacting

https://support.github.com/contact/account
